data/ourCrewData.js

```diff
@@ -0,0 +1,39 @@
+const OurCrewData = [
```

Comment:
tiny thing: We only start with capital letters for React Components. This should be starting with a lowercase letter.